x-pack/plugins/ml/public/jobs/jobs_list/components/multi_job_actions/group_selector/group_list/group_list.js

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+import PropTypes from 'prop-types';
+import React, {
+  Component,
+} from 'react';
+
+import {
+  EuiIcon,
+} from '@elastic/eui';
+
+import './styles/main.less';
+
+import { JobGroup } from '../../../job_group';
+
+function Check({ group, selectedGroups }) {
+  if (selectedGroups[group.id] !== undefined) {
+    if (selectedGroups[group.id].partial) {
+      return (
+        <div className="check selected">
+          <span>&mdash;</span>
+        </div>
+      );
+    } else {
+      return (
+        <div className="check selected">
+          <EuiIcon type="check" />
+        </div>
+      );
+    }
+  } else {
+    return (
+      <div className="check" />
+    );
+  }
```

Comment:
AFAIK it's best practice to add `return null` if a component should conditionally not render at all https://reactjs.org/docs/conditional-rendering.html#preventing-component-from-rendering

Reply:
I agree with that practice, however this is not just an empty `div` it has a class which supplies necessary styling.
Without it the unselected groups are shifted over to the left:
![image](https://user-images.githubusercontent.com/22172091/43852285-c0bbffa6-9b34-11e8-85ef-6e49019dda21.png)

Reply:
You're right, sorry I misread the code and thought there might be cases where nothing would be returned ...